source/features/latest-tag-button.tsx

```diff
@@ -76,6 +78,28 @@ const getRepoPublishState = cache.function(async (): Promise<RepoPublishState> =
 	cacheKey: () => __featureName__ + ':' + getRepoURL()
 });
 
+function getTagOid(node: AnyObject): string {
+	if (node.tag === {}) {
+		return node.tag.commit.oid;
+	}
+
+	return node.tag.oid;
+}
+
+const getDifference = cache.function(async (latestTag: string): Promise<string> => {
```

Comment:
Maybe `getAheadByCount`. Paging @sindresorhus for better naming

Reply:
I've used getAheadByCount for now - happy to update again if necessary